templates/cloudformation/apps/aws-resources.yaml

```diff
@@ -0,0 +1,10 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Template
+
+Resources:
+  AEMLicense:
```

Comment:
Please add comment that this a dummy resource just to get placeholder working since the license itself is not going to be used within stack builder.

Reply:
Done